wp-content/themes/csisjti/assets/_scss/components/_page-header.scss

```diff
@@ -0,0 +1,150 @@
+@use 'abstracts' as *;
+
+.single__header {
+  position: relative;
+  display: grid;
+  grid-template-areas:
+    'social-share'
+    'category'
+    'title'
+    'subtitle'
+    'desc'
+    'date'
+    'right-content'
+    'caption';
+  width: 100%;
+  min-height: 300px;
+  margin-bottom: rem(64);
+  padding-top: var(--header-height);
+  padding-bottom: rem(24);
+  color: $color-white-190;
+  background: var(--page-header-bg);
+  @include full-width-background(var(--page-header-bg));
+
+  @include breakpoint(large) {
+    grid-template-areas:
+      '. social-share'
+      'category right-content'
+      'title right-content'
+      'subtitle right-content'
+      'desc right-content'
+      'date right-content'
+      '. right-content'
+      '. caption';
+    grid-template-columns: 2fr 1fr;
+    grid-template-rows:
+      max-content minmax(0, max-content) max-content minmax(0, max-content)
+      auto;
+    column-gap: rem(24);
+  }
+
+  .post-meta__categories {
+    grid-area: category;
+    @extend %font-ui-text-md-bold-uppercase-barlow;
+
+    &::after {
+      content: '';
+      display: block;
+      width: 4%;
```

Comment:
I would set this width in pixels or rems rather than percents. It's starts to get really small on mobile.